src/v1/profile/bookShelf/ProfileBookshelfPresenter.tsx

```diff
@@ -0,0 +1,46 @@
+import { APIBookshelf } from '@/types/bookshelf';
+
+import Badge from '@/v1/base/Badge';
+import BookShelf from '@/v1/bookShelf/BookShelf';
+
+import { IconArrowRight, IconHeart } from '@public/icons';
+import Link from 'next/link';
+
+const ProfileBookshelfPresenter = ({
+  bookshelfId,
+  books,
+  likeCount,
+}: APIBookshelf) => {
+  return (
+    <div className="flex flex-col gap-[0.6rem]">
+      <div className="flex items-center justify-between">
+        <h3 className="text-lg font-bold">책장</h3>
+        <div className="flex gap-[2rem]">
+          <Badge colorScheme="red" fontWeight="bold" size="small">
+            <div className="flex items-center gap-[0.4rem]">
+              <IconHeart
+                fill="#F56565"
+                stroke="white"
+                stroke-width={1.5}
+                height="1.3rem"
+                width="1.3rem"
+              />
+              <div className="bold text-xs">{likeCount}</div>
+            </div>
+          </Badge>
```

Comment:
p2;

1. 굉장히 미묘한 차이지만.. `Badge size=medium` 이 아주 살짝 더 크게 스타일링 되어있더라구요.. medium으로 적용하면 디자인과 더 유사할 것 같아요! 😂 더불어 하트 아이콘의 크기를 `1.4rem`으로 살짝 키워주시면 감사하겠습니다..🤓
2. 책장 일부만 보여주는 컴포넌트에서 좋아요 뱃지는 채우지말고 아웃라인만 주자는 얘기를 나눴던 것 같아요! 그리고 `stroke-width`가 아니라 `strokeWidth` 카멜케이스를 사용해야 올바르게 적용될 것 같아요! 관련해서 console에 에러가 떠있더라구요!

```suggestion
          <Badge colorScheme="red" fontWeight="bold" size="medium" isFilled={false}>
            <div className="flex items-center gap-[0.4rem]">
              <IconHeart
                fill="#F56565"
                height="1.4rem"
                width="1.4rem"
              />
              <div className="bold text-xs">{likeCount}</div>
            </div>
          </Badge>
```